client/reader/list-stream/missing.jsx

```diff
@@ -40,6 +41,7 @@ class ListMissing extends React.Component {
 
 		return (
 			<div>
```

Comment:
I think this should probably be a `ReaderMain` so we get the right positioning of the 'missing list' page. I also notice we have a grey background rather than the usual white one as shown in this missing site page:

<img width="595" alt="Screen Shot 2021-06-01 at 12 29 11" src="https://user-images.githubusercontent.com/17325/120251399-83cced00-c2d5-11eb-84f3-29b32d4bc4b3.png">

Reply:
Fixed! I made a couple of changes to `ReaderListManage` in order to ensure there's not two duplicate components of `ReaderMain` on the same page.